docs/Customizing Look and Functionality/Customizing the Template/index.md

```diff
@@ -113,28 +113,28 @@ RGBColor white = new RGBColor(249, 251, 254);
 RGBColor grey = new RGBColor(186, 198, 210);
 RGBColor darkGrey = new RGBColor(57, 78, 96);
 
-TemplateColorScheme dayColorScheme = new TemplateColorScheme();
-dayColorScheme.setBackgroundColor(white);
-dayColorScheme.setPrimaryColor(green);
-dayColorScheme.setSecondaryColor(grey);
+TemplateColorScheme dayColorScheme = new TemplateColorScheme()
+    .setBackgroundColor(white)
+    .setPrimaryColor(green)
+    .setSecondaryColor(grey);
 builder.setDayColorScheme(dayColorScheme);
 
-TemplateColorScheme nightColorScheme = new TemplateColorScheme();
-nightColorScheme.setBackgroundColor(white);
-nightColorScheme.setPrimaryColor(green);
-nightColorScheme.setSecondaryColor(darkGrey);
-
-SetDisplayLayout setDisplayLayout = new SetDisplayLayout(PredefinedLayout.GRAPHIC_WITH_TEXT.toString());
-setDisplayLayout.setDayColorScheme(dayColorScheme);
-setDisplayLayout.setNightColorScheme(nightColorScheme);
-setDisplayLayout.setOnRPCResponseListener(new OnRPCResponseListener() {
-    @Override
-    public void onResponse(int correlationId, RPCResponse response) {
-        if (response.getSuccess()){
-            // Success
+TemplateColorScheme nightColorScheme = new TemplateColorScheme()
+    .setBackgroundColor(white)
+    .setPrimaryColor(green)
+    .setSecondaryColor(darkGrey);
+
+SetDisplayLayout setDisplayLayout = new SetDisplayLayout(PredefinedLayout.GRAPHIC_WITH_TEXT.toString())
+    .setDayColorScheme(dayColorScheme)
+    .setNightColorScheme(nightColorScheme)
+    .setOnRPCResponseListener(new OnRPCResponseListener() {
```

Comment:
This wouldn't compile because `setOnRPCResponseListener()` returns type of parent class. This applies to all setters in parent classes in this PR.